doc/write_your_own/components/Workflow1.ipynb

```diff
@@ -318,40 +277,21 @@
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {
-    "collapsed": false,
     "urth": {
      "dashboard": {}
     }
    },
-   "outputs": [],
    "source": [
-    "if dim_y.value in coordinates:\n",
-    "    coordinates.remove(dim_y.value)\n",
-    "\n",
-    "index = []\n",
-    "for i in range(len(coordinates)):\n",
-    "    coord_index = ipywidgets.IntSlider(\n",
-    "        value=0,\n",
-    "        min=0,\n",
-    "        max=len(base_cube.coord(coordinates[i]).points-1),\n",
-    "        step=1,\n",
-    "        description='Index of ' + coordinates[i])\n",
-    "    index.append(coord_index)\n",
-    "    IPython.display.display(index[i])"
+    "# 3. Choose your formatting options"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {
-    "urth": {
-     "dashboard": {}
-    }
-   },
+   "metadata": {},
    "source": [
-    "# 3. Choose your formatting options"
+    "Construct a set of widgets to present options for plot customization.  Some of the options are only applicable to certain types of plots, so these widgets are dependant on the value of the plot-type widget."
```

Comment:
The last code cell (i.e. the code cell after this text cell) is very long! I wonder whether it would be good to split it into the bit that provides all the formatting options and the other bit that produces the display elements (so split after L44: `formatting = [...`).